src/components/DataProvider/index.js

```diff
@@ -739,6 +745,9 @@ DataProvider.propTypes = {
   // loaderConfig => void
   // called whenever data is fetched by the loader
   onFetchData: PropTypes.func,
+  // error => void
```

Comment:
What do you think about `(error, params) => void`?

Reply:
Haha sorry I think I merged at the same time as you were commenting.
Params as in the ones passed into the loader?

``` 
loaderResult = await loader({
        id,
        timeDomain,
        timeSubDomain,
        pointsPerSeries,
        oldSeries: seriesObject,
        reason,
      });
```